roles/edpm_libvirt/tasks/configure.yml

```diff
@@ -144,15 +144,15 @@
   become: true
   loop:
     - {"src": "{{ edpm_libvirt_tls_cert_src_dir }}/tls.crt", "dest": "/etc/pki/libvirt/servercert.pem"}
-    - {"src": "{{ edpm_libvirt_tls_cert_src_dir }}/tls.key", "dest": "/etc/pki/libvirt/private/serverkey.pem"}
+    - {"src": "{{ edpm_libvirt_tls_cert_src_dir }}/tls.key", "dest": "/etc/pki/libvirt/private/serverkey.pem", "mode": "0600"}
     - {"src": "{{ edpm_libvirt_tls_cert_src_dir }}/tls.crt", "dest": "/etc/pki/libvirt/clientcert.pem"}
     - {"src": "{{ edpm_libvirt_tls_cert_src_dir }}/tls.key", "dest": "/etc/pki/libvirt/private/clientkey.pem"}
```

Comment:
do we need to set 600 on this private key too?

Reply:
no. That is not virt-pki-validate recommendation.

Reply:
OK. I won't block on this. But lets wait for @SeanMooney to approve it.

Reply:
access to the host is privileged. so to access this key you would already have access to a privileged security context (i.e. a shell on the host)
I agree that 600 would minimally improve the security posture but I hesitate to deviate from the tested defaults recommended by the libvirt project.